### PR TITLE
[Android] Change default of ram max used idle frac to 0.9 like pc.

### PR DESCRIPTION
**Pull request by talregev**
Change default of ram max used idle frac to 0.9 like pc.

### CODE REVIEW

File: lib/prefs.cpp [hunk 1]
```diff
@@ -234,7 +234,7 @@ void GLOBAL_PREFS::defaults() {
 #endif
     ram_max_used_busy_frac = 0.5;
 #ifdef ANDROID
-    ram_max_used_idle_frac = 0.5;
+    ram_max_used_idle_frac = 0.9;
```
Comment (adamradocz) [inline lib/prefs.cpp:237]: I think the code would be simpler if you just simply delete the android section.

Comment (talregev) [reply lib/prefs.cpp:237]: @adamradocz 
Yes, I agree. But I was waiting to hear the opinion of @AenBleidd about changing the value itself.
If he agree, I will change the code to be more simpler. 

@adamradocz Thank you for your time to review my PRs.

**Comment by AenBleidd** [comment]
Android devices (here I talk about smartphones only) usually have no so much RAM. If set RAM usage by BOINC to 90% will likely remove system UI processes from RAM to swap that will lead to lugging UI when turning screen on and unblocking the phone. If user is experienced and they know what they are doing - they could easily set more comfortable RAM usage for them.
I tend to left this settings as-is and not merge this PR.
@talregev, if you have strong arguments about this change - feel free to put them here.

**Comment by talregev** [comment]
@AenBleidd I don't have strong argument about that. I will close this PR.

**Comment by adamradocz** [comment]
This is idle setting, so I assume when the screen is turned on again, the tasks stop and the user won't notice anything about it, maybe a little lag. But the extra RAM could make a significant impact in terms of computing.
BTW, I don't remember this option is editable in the advanced menu?

**Comment by talregev** [comment]
It editable and user can change this setting.

**Pull request closed without merging**